python/ray/tune/tests/test_tune_restore.py

```diff
@@ -17,10 +18,11 @@
 import ray
 from ray import tune
 from ray._private.test_utils import recursive_fnmatch, run_string_as_driver
-from ray.air._internal.checkpoint_manager import _TrackedCheckpoint, CheckpointStorage
 from ray.train import CheckpointConfig
+from ray.train._checkpoint import Checkpoint
```

Comment:
```suggestion
from ray.train import Checkpoint
```

Reply:
Is this reference updated? Last time I used this it was still pointing to the AIR Checkpoint.

Can we clean this up when we get rid of the storage path check?

Reply:
Ah it is gated [here](https://github.com/ray-project/ray/blob/55e8dab1bbb5971d09a42a8bb660f5b3424b2e6d/python/ray/train/__init__.py#L25-L28).

I also have an open PR [here](https://github.com/ray-project/ray/pull/38951) which further cleans up a lot of the usage for this.